diff --git a/node_modules/@blocksuite/data-view/dist/core/filter/literal/define.js b/node_modules/@blocksuite/data-view/dist/core/filter/literal/define.js
index 3da8ca1..fcb527a 100644
--- a/node_modules/@blocksuite/data-view/dist/core/filter/literal/define.js
+++ b/node_modules/@blocksuite/data-view/dist/core/filter/literal/define.js
@@ -1,6 +1,6 @@
 import { menu } from '@blocksuite/affine-components/context-menu';
 import { unsafeCSSVarV2 } from '@blocksuite/affine-shared/theme';
-import { CheckBoxCkeckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
+import { CheckBoxCheckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
 import { html } from 'lit';
 import { t } from '../../logical/type-presets.js';
 import { createLiteral } from './create.js';
@@ -83,7 +83,7 @@ export const allLiteralConfig = [
                     items: type.element.data?.map(tag => {
                         const selected = set.has(tag.id);
                         const prefix = selected
-                            ? CheckBoxCkeckSolidIcon({ style: `color:#1E96EB` })
+                            ? CheckBoxCheckSolidIcon({ style: `color:#1E96EB` })
                             : CheckBoxUnIcon();
                         return menu.action({
                             name: tag.value,
diff --git a/node_modules/@blocksuite/data-view/dist/core/group-by/renderer/boolean-group.js b/node_modules/@blocksuite/data-view/dist/core/group-by/renderer/boolean-group.js
index ff7b3b2..43ddca9 100644
--- a/node_modules/@blocksuite/data-view/dist/core/group-by/renderer/boolean-group.js
+++ b/node_modules/@blocksuite/data-view/dist/core/group-by/renderer/boolean-group.js
@@ -1,4 +1,4 @@
-import { CheckBoxCkeckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
+import { CheckBoxCheckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
 import { css, html } from 'lit';
 import { BaseGroup } from './base.js';
 export class BooleanGroupView extends BaseGroup {
@@ -15,7 +15,7 @@ export class BooleanGroupView extends BaseGroup {
     render() {
         return html ` <div class="data-view-group-title-boolean-view">
       ${this.value
-            ? CheckBoxCkeckSolidIcon({ style: `color:#1E96EB` })
+            ? CheckBoxCheckSolidIcon({ style: `color:#1E96EB` })
             : CheckBoxUnIcon()}
     </div>`;
     }
diff --git a/node_modules/@blocksuite/data-view/dist/property-presets/checkbox/cell-renderer.js b/node_modules/@blocksuite/data-view/dist/property-presets/checkbox/cell-renderer.js
index a1a8ed2..ea84d8e 100644
--- a/node_modules/@blocksuite/data-view/dist/property-presets/checkbox/cell-renderer.js
+++ b/node_modules/@blocksuite/data-view/dist/property-presets/checkbox/cell-renderer.js
@@ -33,7 +33,7 @@ var __runInitializers = (this && this.__runInitializers) || function (thisArg, i
     return useValue ? value : void 0;
 };
 import { unsafeCSSVarV2 } from '@blocksuite/affine-shared/theme';
-import { CheckBoxCkeckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
+import { CheckBoxCheckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
 import { css, html } from 'lit';
 import { query } from 'lit/decorators.js';
 import { BaseCellRenderer } from '../../core/property/index.js';
@@ -120,7 +120,7 @@ let CheckboxCell = (() => {
         render() {
             const checked = this.value ?? false;
             const icon = checked
-                ? CheckBoxCkeckSolidIcon({ style: `color:#1E96EB` })
+                ? CheckBoxCheckSolidIcon({ style: `color:#1E96EB` })
                 : CheckBoxUnIcon();
             return html ` <div class="affine-database-checkbox-container">
       <div
diff --git a/node_modules/@blocksuite/data-view/dist/view-presets/table/pc/row/row-select-checkbox.js b/node_modules/@blocksuite/data-view/dist/view-presets/table/pc/row/row-select-checkbox.js
index 8c599a5..4d5841a 100644
--- a/node_modules/@blocksuite/data-view/dist/view-presets/table/pc/row/row-select-checkbox.js
+++ b/node_modules/@blocksuite/data-view/dist/view-presets/table/pc/row/row-select-checkbox.js
@@ -34,7 +34,7 @@ var __runInitializers = (this && this.__runInitializers) || function (thisArg, i
 };
 import { ShadowlessElement } from '@blocksuite/block-std';
 import { SignalWatcher, WithDisposable } from '@blocksuite/global/utils';
-import { CheckBoxCkeckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
+import { CheckBoxCheckSolidIcon, CheckBoxUnIcon } from '@blocksuite/icons/lit';
 import { computed } from '@preact/signals-core';
 import { css, html } from 'lit';
 import { property } from 'lit/decorators.js';
@@ -122,7 +122,7 @@ let RowSelectCheckbox = (() => {
             return html `
       <div class="${classString}">
         ${this.isSelected$.value
-                ? CheckBoxCkeckSolidIcon({ style: `color:#1E96EB` })
+                ? CheckBoxCheckSolidIcon({ style: `color:#1E96EB` })
                 : CheckBoxUnIcon()}
       </div>
     `;
