diff --git a/node_modules/@blocksuite/affine-components/dist/context-menu/button.js b/node_modules/@blocksuite/affine-components/dist/context-menu/button.js
index 8d24a87..69c970b 100644
--- a/node_modules/@blocksuite/affine-components/dist/context-menu/button.js
+++ b/node_modules/@blocksuite/affine-components/dist/context-menu/button.js
@@ -33,7 +33,7 @@ var __runInitializers = (this && this.__runInitializers) || function (thisArg, i
     return useValue ? value : void 0;
 };
 import { IS_MOBILE } from '@blocksuite/global/env';
-import { CheckBoxCkeckSolidIcon, CheckBoxUnIcon, DoneIcon, } from '@blocksuite/icons/lit';
+import { CheckBoxCheckSolidIcon, CheckBoxUnIcon, DoneIcon, } from '@blocksuite/icons/lit';
 import { cssVarV2 } from '@toeverything/theme/v2';
 import { css, html, unsafeCSS } from 'lit';
 import { property } from 'lit/decorators.js';
@@ -246,7 +246,7 @@ export const menuButtonItems = {
         const data = {
             content: () => html `
           ${config.checked.value
-                ? CheckBoxCkeckSolidIcon({ style: `color:#1E96EB` })
+                ? CheckBoxCheckSolidIcon({ style: `color:#1E96EB` })
                 : CheckBoxUnIcon()}
           <div class="affine-menu-action-text">
             ${config.label?.() ?? config.name}
